feat: a lot more than just sqlite

tjdevries opened the pull request.

tjdevries committed:
wip: start iterating on sqlite version

tjdevries committed:
[WIP]: Thu Oct 31 12:40:38 PM EDT 2024

tjdevries committed:
[WIP]: Thu Oct 31 12:43:46 PM EDT 2024

tjdevries committed:
[WIP]: Thu Oct 31 01:05:58 PM EDT 2024

tjdevries committed:
[WIP]: Thu Oct 31 01:32:19 PM EDT 2024

tjdevries committed:
[WIP]: Thu Oct 31 03:09:02 PM EDT 2024

tjdevries committed:
[WIP]: Thu Oct 31 03:26:25 PM EDT 2024

tjdevries commented:
not ready yet, btw

tjdevries committed:
[WIP]: Thu Oct 31 03:31:05 PM EDT 2024

tjdevries committed:
[WIP]: Thu Oct 31 10:55:37 PM EDT 2024

tjdevries committed:
[WIP]: Thu Oct 31 11:07:44 PM EDT 2024

tjdevries committed:
[WIP]: Thu Oct 31 11:29:01 PM EDT 2024

tjdevries commented:
ok, maybe ready now

emilpriver commented:
LGTM, Thank you for doing this work :D

emilpriver commented:
Wow, thank you for your first pr!

emilpriver commented:
ok I just did that for the meme.

Anywho, Really nice job and thank you for putting in the time to build DBCaml! :D This changes I really neat

emilpriver commented:
Let's make a good db library, which is not hard if we compare stuff to the JS, those are actually shit

emilpriver commented:
I do see that the CI is broken. I currently looking into it

emilpriver committed:
Updated ci to use dune instead of opam

We had some issues where opam tried to make installations and we now
it's wak so now we changed to dune

emilpriver committed:
CI test

emilpriver commented:
This pr:
Closes https://github.com/dbcaml/dbcaml/issues/40
Closes https://github.com/dbcaml/dbcaml/issues/5

emilpriver committed:
Test for the CI

emilpriver committed:
Updated ci name

emilpriver commented:
the CI tests seems to be working now again, it failed due to some missing name in the dune files tho, but we can fix that later :D

tjdevries committed:
[WIP]: Fri Nov  1 01:58:01 PM EDT 2024

tjdevries committed:
[WIP]: Fri Nov  1 02:25:08 PM EDT 2024

tjdevries committed:
[WIP]: Wed Nov 13 09:27:17 PM EST 2024
